src/instructlab/eval/pr_bench_generator.py

```diff
@@ -0,0 +1,114 @@
+# Standard
+import json
+import os
+import time
+
+# Third Party
+from tqdm import tqdm
+import git
+import shortuuid
+import yaml
+
+# Local
+from .common import bench_dir
+
+
+def get_file_paths(directory):
+    file_paths = []
+    for root, _, files in os.walk(directory):
+        for file in files:
+            if file.split("/")[-1] == "qna.yaml":
+                file_paths.append(os.path.join(root, file))
+    return file_paths
+
+
+def read_qna(fn):
+    with open(fn, "r", encoding="utf-8") as file:
+        contents = yaml.safe_load(file)
+    return contents.get("seed_examples")
```

Comment:
Are we checking the YAML/schema validity at all?

Don't think we need for this PR specifically, but @bjhargrave is working on a Dev Doc to get this functionality into the `instructlab-schema` package: https://github.com/instructlab/dev-docs/pull/101